kms/apiv1/requests.go

```diff
@@ -173,13 +175,14 @@ type CreateKeyResponse struct {
 
 // CreateSignerRequest is the parameter used in the kms.CreateSigner method.
 type CreateSignerRequest struct {
-	Signer        crypto.Signer
-	SigningKey    string
-	SigningKeyPEM []byte
-	TokenLabel    string
-	PublicKey     string
-	PublicKeyPEM  []byte
-	Password      []byte
+	Signer           crypto.Signer
+	SigningKey       string
+	SigningKeyPEM    []byte
+	TokenLabel       string
+	PublicKey        string
+	PublicKeyPEM     []byte
+	Password         []byte
+	PasswordPrompter func() (string, pemutil.PasswordPrompter)
```

Comment:
We should redefine or alias PasswordPrompter in this package.

I think we should simplify this to `PasswordPrompter PasswordPrompter`. And then do something like `pemutil.WithPasswordPromp("Please ....", req.PasswordPrompter)`. If some code wants a different one they can also ignore the default prompt.